CMake/External_PROJ.cmake

```diff
@@ -1,11 +1,23 @@
 
+if(fletch_ENABLE_SQLite3)
+  get_system_library_name(sqlite3 sqlite_libname)
+  set(PROJ_ARGS_SQLite3
+    -DSQLite3_DIR:PATH=${fletch_BUILD_INSTALL_PREFIX}/share/cmake
+    )
+  list(APPEND PROJ_DEPENDS SQLite3)
+else()
+  find_package(SQLite3 REQUIRED)
```

Comment:
Should this then pass the `SQLite3_DIR`?

Reply:
This find_package is for a system SQLite if fletch isn't building it so probably not, since we wouldn't know what to pass